src/core/worker.c

```diff
@@ -93,6 +97,14 @@ QuicWorkerInitialize(
         goto Error;
     }
 
+    Worker->ExecutionContext.Context = Worker;
+    Worker->ExecutionContext.Callback = QuicWorkerLoop;
```

Comment:
Is Worker->ExecutionContext.Callback used when QUIC_USE_EXECUTION_CONTEXTS is not defined? I ask because you hid most shared-EC code behind ifdefs but not this part, and because QuicWorkerThread already calls QuicWorkerLoop in the non-shared-EC version.

Reply:
Yes, I believe more _could_ be hidden behind the #ifdefs but I'm still not 100% sure we want this compile-time in the end. Thoughts on that?

Reply:
Right, this is partly related to my other comment.